INSTALL.md

```diff
@@ -208,7 +206,8 @@ sudo apt-get install -y autoconf automake libtool pkg-config libfuse3-dev \
 On RHEL or derivatives:
 
 ```sh
-sudo dnf install -y fuse3-devel lzo-devel lz4-devel
+sudo dnf install -y fuse3-devel zlib-devel lzo-devel lz4-devel \
+    xz-devel libzstd-devel
```

Comment:
These 3 additional libraries were not explicitly listed because they're already included by
```
dnf groupinstall -y 'Development Tools'
```
So I don't think they need to be here or in scripts/ci-rpm-build-test.

Reply:
Seems a bit arbitrary to me, hope they don't change it then. But OK, will leave as it was.